.github/workflows/release.yaml

```diff
@@ -59,24 +59,9 @@ jobs:
 
           # create tarball
           cd ./output && tar -cvzf manifests.tar.gz $files
-      - name: Create release
```

Comment:
Wait, was this intentional? Or would goreleaser create the release instead?

Reply:
Yes goreleaser will create it

Reply:
Got it, thanks for the clarifications.